<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EWASiteWidgetSettingsDlgUi</class>
 <widget class="QWidget" name="EWASiteWidgetSettingsDlgUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>182</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>108</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="pSiteWidgetSettingsStackedWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pSiteWidgetGeometryPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Geometry</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="widthSpinBox">
            <property name="toolTip">
             <string>Popup message width (pixels)</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="value">
             <number>265</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="heigtSpinBox">
            <property name="toolTip">
             <string>Popup message heigth (pixels)</string>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="value">
             <number>170</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Horizontal:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="scrXSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Vertical:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="scrYSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="EWASiteWidgetColorManager" name="pXMsgColorManager" native="true"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Always show on Title Bar</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="backCheckBox">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ewa.qrc">
              <normaloff>:/images/skip_backward.png</normaloff>:/images/skip_backward.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="refreshCheckBox">
            <property name="toolTip">
             <string>Visibility of &lt;b&gt;Reload&lt;/b&gt;/&lt;b&gt;Stop&lt;/b&gt; button</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ewa.qrc">
              <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="forwardCheckBox">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ewa.qrc">
              <normaloff>:/images/skip_forward.png</normaloff>:/images/skip_forward.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="titleCheckBox">
            <property name="toolTip">
             <string>Visibility of &lt;b&gt;Caption lable&lt;/b&gt;</string>
            </property>
            <property name="text">
             <string>[Title]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="stickCheckBox">
            <property name="toolTip">
             <string>Visibility of &lt;b&gt;Stick/Unstick&lt;/b&gt; button</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ewa.qrc">
              <normaloff>:/images/stick.png</normaloff>:/images/stick.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="maxminCheckBox">
            <property name="toolTip">
             <string>Visibility of &lt;b&gt;Max-/Minimize&lt;/b&gt; button</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ewa.qrc">
              <normaloff>:/images/big.png</normaloff>:/images/big.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="closeCheckBox">
            <property name="toolTip">
             <string>Visibility of &lt;b&gt;Close&lt;/b&gt; button</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ewa.qrc">
              <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pWebViewGeometryPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Page view size - like as browser window size</string>
         </property>
         <property name="title">
          <string>Geometry</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="pageWidthSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="pageHeightSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Position of shown page region</string>
         </property>
         <property name="title">
          <string>View Point</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Horizontal:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="msgXSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Vertical:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="msgYSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="contentsSizeLabel">
           <property name="text">
            <string>Contents size: Uncknown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="adjustSizeButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Adjust size</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="ewa.qrc">
             <normaloff>:/images/accept.png</normaloff>:/images/accept.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EWASiteWidgetColorManager</class>
   <extends>QWidget</extends>
   <header>ewa_sitewidgetcolormanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ewa.qrc"/>
 </resources>
 <connections/>
</ui>
